c-api/src/comment.rs

```diff
@@ -1,12 +1,12 @@
 use super::*;
 
 #[no_mangle]
-pub extern "C" fn lol_html_comment_text_get(comment: *const Comment) -> Str {
+pub unsafe extern "C" fn lol_html_comment_text_get(comment: *const Comment) -> Str {
```

Comment:
Introducing this change, let's set `#![forbid(unsafe_op_in_unsafe_fn)]` for the crate

Reply:
Not ready to do that yet, as it will cause merge conflicts for the streaming handlers branch.